addons/rating/models/rating_mixin.py

```diff
@@ -172,9 +172,9 @@ def rating_apply(self, rate, token=None, feedback=None, subtype=None):
         :type subtype : string
         :returns rating.rating record
         """
-        Rating, rating = self.env['rating.rating'], None
+        Rating, rating = self.env['rating.rating'].sudo(), None
         if token:
-            rating = self.env['rating.rating'].search([('access_token', '=', token)], limit=1)
+            rating = Rating.search([('access_token', '=', token)], limit=1)
```

Comment:
(Maybe) Same here as it will allow to write https://github.com/odoo/odoo/blob/fd6c726539597c666a43d2f97f0631455979520a/addons/rating/models/rating_mixin.py#L181